ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -106,6 +106,13 @@ io::log_yellow "Verify googleapis commit hash for generated code"
   ./ci/check-generated-code-hash.sh
 )
 
+echo "================================================================"
+io::log_yellow "Verify generator golden file md5 hashes"
+(
+  cd "${PROJECT_ROOT}"
```

Comment:
Do we need this if `check-golden-md5-hashes.sh` is going to heed `${PROJECT_ROOT}` itself (or vice versa)?